ci/deploy-deb.sh

```diff
@@ -1,21 +1,23 @@
 #!/bin/bash
 
+set -e
```

Comment:
Added so that the CI can just exit if an error is encountered, preventing the script from trying to continue to run.